pyinjective/core/broadcaster.py

```diff
@@ -157,6 +172,10 @@ async def broadcast(self, messages: List[any_pb2.Any]):
 
         messages_for_transaction = self._account_config.messages_prepared_for_transaction(messages=messages)
 
+        # before constructing the transaction, check if sender address is in the OFAC list
+        if self._account_config.trading_injective_address in self._ofac_list:
+            raise BannedAddressError(f"Address {self._account_config.trading_injective_address} is in the OFAC list")
+
```

Comment:
Instead of doing the check when trying to broadcast a transaction, I think it is better to do that in the braodcaster instance creation (in the `__init__` method), to try to fail fast.

Reply:
there are two approaches to exception handling: 1) creating some custom exction classes 2) throwing an exception which is constructed on the spot from some default `Exception` constructor. Since there was a file with expcetion classes, I tried to introduce my own custom exception, which is approach number 1). Should I proceed with approach number 2)?

Reply:
Removed all the custom exceptions for now.